examples/src/app/equal-to-validator/equal-to-validator.component.ts

```diff
@@ -0,0 +1,33 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { email, Validator } from '../items';
+import { ActivatedRoute } from '@angular/router';
+
+@Component({
+  selector: 'app-equals-to-field-validator',
+  templateUrl: './equal-to-validator.component.html'
+})
+export class EqualToValidatorComponent implements OnInit, OnDestroy {
+
+  private sub: any;
+  emailItems = email;
+  item: string = this.emailItems[0].name;
```

Comment:
Can you please rename the variable `emailItems `

Reply:
sure